local_build.py

```diff
@@ -168,6 +170,10 @@ def build_authorea_latex(localdir, builddir, latex_exec, bibtex_exec, outname,
                 titlestr = f.read()
         titlecontent.append(r'\title{' + titlestr + '}')
 
+    authorcontent = ""
+    with open(os.path.join(localdir, 'authors.tex')) as f:
```

Comment:
I'm confused by this: why are you opening the file?  It should be sufficient just to do the ``get_input_string`` line below, as then LaTeX will drop the content in there instead of you needing to access the file.

So perhaps instead you want this to be ``if os.path.exists(os.path.join(localdir, 'authors.tex')):``?  Then if the files is present it will be used, but otherwise just the empty string from above will get in?